File: src/main/java/com/martinodutto/tpt/controllers/entities/SetResult.java
package com.martinodutto.tpt.controllers.entities;

public class SetResult {

    private int firstPlayerGames;

    private int secondPlayerGames;

    public int getFirstPlayerGames() {
        return firstPlayerGames;
    }

    public void setFirstPlayerGames(int firstPlayerGames) {
        this.firstPlayerGames = firstPlayerGames;
    }

    public int getSecondPlayerGames() {
        return secondPlayerGames;
    }

    public void setSecondPlayerGames(int secondPlayerGames) {
        this.secondPlayerGames = secondPlayerGames;
    }

    @Override
    public String toString() {
        return "SetResult{" +
                "firstPlayerGames=" + firstPlayerGames +
                ", secondPlayerGames=" + secondPlayerGames +
                '}';
    }
}

File: src/main/apache/src/app/model/set-result.ts
export class SetResult {

  private _firstPlayerGames: number;

  private _secondPlayerGames: number;

  get firstPlayerGames(): number {
    return this._firstPlayerGames;
  }

  set firstPlayerGames(value: number) {
    this._firstPlayerGames = value;
  }

  get secondPlayerGames(): number {
    return this._secondPlayerGames;
  }

  set secondPlayerGames(value: number) {
    this._secondPlayerGames = value;
  }
}
